packs/data/the-slithering-bestiary.db/fuming-sludge.json

```
@@ -70,9 +82,21 @@
                     }
                 },
                 "description": {
-                    "value": "<p>[[/r 1d8[persistent,fire]]] damage</p>"
+                    "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "category": "persistent",
+                        "damageType": "fire",
+                        "diceNumber": 1,
+                        "dieSize": "d8",
+                        "key": "DamageDice",
+                        "predicate": [
+                            "item:slug:ejected-glob"
+                        ],
+                        "selector": "strike-damage"
```

```suggestion
                        "selector": "strike-damage"
```
Since it's on all its strikes anyways may as well cut down the number of REs by one.